pkgs/servers/sql/mariadb/default.nix

```diff
@@ -201,12 +201,12 @@ server = stdenv.mkDerivation (common // {
     chmod +x "$out"/bin/wsrep_sst_common
     rm "$out"/bin/{mariadb-client-test,mariadb-test,mysql_client_test,mysqltest}
     rm -r "$out"/data # Don't need testing data
-    mv "$out"/OFF/suite/plugins/pam/pam_mariadb_mtr.so "$out"/share/pam/lib/security
-    mv "$out"/OFF/suite/plugins/pam/mariadb_mtr "$out"/share/pam/etc/security
-    rm -r "$out"/OFF
   '' + optionalString withStorageMroonga ''
     mv "$out"/share/{groonga,groonga-normalizer-mysql} "$out"/share/doc/mysql
   '' + optionalString (!stdenv.hostPlatform.isDarwin) ''
+    mv "$out"/OFF/suite/plugins/pam/pam_mariadb_mtr.so "$out"/share/pam/lib/security
+    mv "$out"/OFF/suite/plugins/pam/mariadb_mtr "$out"/share/pam/etc/security
+    rm -r "$out"/OFF
```

Comment:
I don't expect we want to keep `"$out"/OFF` on Darwin, right?

Reply:
Sounds right to me.

Reply:
It doesn't actually seem to be created on darwin, so the build fails because of that now:
```
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/policy/apparmor/usr.sbin.mysqld.local
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/policy/selinux
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/policy/selinux/README
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/policy/selinux/mariadb-server.te
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/policy/selinux/mariadb-server.fc
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/policy/selinux/mariadb.te
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/lib/pkgconfig/mariadb.pc
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/aclocal/mysql.m4
  -- Installing: /nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/share/doc/mysql/mysql.server
  rm: cannot remove '/nix/store/2f7mg8dm4xh7hk2vvvdadia7gwaa785f-mariadb-server-10.4.13/OFF': No such file or directory
cannot build derivation '/nix/store/zap5nxyb8kkdldla5hi9hyb4hn15f2gj-diesel-cli-1.4.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7jxmgixk4ax0jm07hvaqsb2r25fbd7sj-env.drv': 2 dependencies couldn't be built
[0 built (1 failed), 202 copied (2112.6 MiB), 572.0 MiB DL]
error: build of '/nix/store/7jxmgixk4ax0jm07hvaqsb2r25fbd7sj-env.drv' failed
https://github.com/NixOS/nixpkgs/pull/90139
2 packages failed to build:
diesel-cli mysql
```

Reply:
OK, ofborg seems to fail to build Darwin PRs so thanks :+1:.